content/docs/resources/standard-routes.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Standard Routes"
+weight: 7
+---
+
+### Cases
+
+{{< route method="POST" route="/gossip/v1/cases" >}}
+
+This endpoint notifies the server of a new case
+
+Request Body:
+
+| Field      | Type.       | Description                                                          |
+| ---------- | ----------- | -------------------------------------------------------------------- |
+| `guild`    | snowflake   | Guild ID where the case took place.                                  |
+| `user`     | snowflake   | User ID where the action was taken.                                  |
+| `actioner` | snowflake   | User ID that created the case. Might be bot ID if it was automated.  |
+| `action`   | case_action | Type of action taken.                                                |
+| `duration` | long.       | Milliseconds for duration. 0 if permanent                            |
+| `reason`   | string.     | Reason for action, might be empty                                    |
+
+
+Where `case_action` is one of (`BAN`, `KICK`, `MUTE`, `WARN`)
```

Comment:
I think this should be an open-ended field and the server returns a 400 bad request if an invalid case type is given

Reply:
Technically it is, we are just defining some standard ones and it could be opened more if both support it

Reply:
For the standard ones in that case I think we should also add UNMUTE, UNBAN, NOTE

Reply:
done